tests/state/dependencies/modules/defarguments_module1/__init__.py

```diff
@@ -41,5 +41,7 @@ def register(mf):
         "var5": var5_fn,
         "var6": var6_fn,
         "annotation": lambda: 10,
-        "annotation.bug": lambda: 10
+        "annotation.bug": lambda: 10,
+        "lambda_with_comment": lambda: 10,  # just some comment
+        "var_with_comment": 10,  # just some comment
```

Comment:
These tests need their own module, since this is a module which only tests `def` as an argument.

Reply:
Agreed.
Moved to seperate module `potential_ast_bugs_module`.